bin/i18n/resources/apps/labs.rb

```diff
@@ -14,6 +14,8 @@ def self.sync_in
           puts 'Preparing *labs files'
           FileUtils.mkdir_p(I18N_SOURCE_DIR_PATH)
 
+          FileUtils.cp(CDO.dir('apps/node_modules/@code-dot-org/ml-activities/i18n/oceans.json'), File.join(I18N_SOURCE_DIR_PATH, 'fish.json'))
```

Comment:
As `oceans.json` comes from an external repo, it would be more appropriate to move it to `external_sources.rb` instead of `labs.rb`.

Reply:
I'm not sure about it, because since the i18n source folder for this file is `blockly-mooc` (labs) and not `external_sources`, it would be confusing

Reply:
I'm not sure about this, because since the i18n source folder fort he file is `blockly-mooc` (labs) and not `external_sources`, it would be confusing

Reply:
I'm not sure about this, because it would be confusing, since the i18n source folder for the file is `blockly-mooc` (labs) and not `external_sources`

Reply:
posting this here for visibility:
We needed `external-sources` for `mlPlayground` because we had two `mlPlayground` files (one from the external repo and one from apps). However, I think it is more clear having all external repo files in external sources moving on.